src/test/unit/algorithms/ConnectionsTest.cpp

```diff
@@ -227,16 +227,17 @@ namespace {
     // Add an additional synapse over the limit
     presynapticCell.idx = 52;
     synapse = connections.createSynapse(segment, presynapticCell, 0.52);
+    EXPECT_LT(synapse.idx, 2);
```

Comment:
I think EXPECT makes sense. In today's code it would fail this EXPECT but the remainder will pass. It doesn't hurt to have that information.

Ok, I can use EXPECT_EQ here. In the other test `DestroySynapsesThenReachLimit` I'll keep using EXPECT_LT because the behavior is less defined (there's a tie in the permanences) and we're really just making sure it doesn't continue growing the synapses vector like it does today.